core/store/migrate/migrations/0179_ocr2_multiple_configs_per_spec.sql

```diff
@@ -0,0 +1,11 @@
+-- +goose Up
+ALTER TABLE ocr2_contract_configs
+  ADD COLUMN plugin_id INTEGER NOT NULL DEFAULT 0,
+  DROP CONSTRAINT offchainreporting2_contract_configs_pkey,
+  ADD CONSTRAINT ocr2_contract_configs_unique_id_pair UNIQUE (ocr2_oracle_spec_id, plugin_id);
```

Comment:
@reductionista can you check my SQL here please? I'm removing spec ID as a primary key to allow for multiple plugin_ids per spec_id.
There's also a foreign key constraint with offchainreporting2_oracle_specs(id) - I hope that's not a problem, right?